### PR TITLE
[Redesign] Add contact page

### PR DESCRIPTION
**Pull request by joelverhagen**
1. Add contact page.
1. Add new HTML helper for checkbox.
1. Add common heading and subheading CSS. Copied from https://github.com/NuGet/NuGetGallery/pull/4048.

Address https://github.com/NuGet/NuGetGallery/issues/4007.

/cc @microsoftsam

### CODE REVIEW

**Comment by joelverhagen** [comment]
# Logged out
screencapture-localhost-policies-contact-1496865003503

# Logged in
screencapture-localhost-policies-contact-1496865017207

**Comment by loic-sharma** [comment]
Just FYI, this pull request is currently set to merge into `jver-polish` - I'm not sure if that's intentional or not.

**Comment by joelverhagen** [comment]
> Just FYI, this pull request is currently set to merge into `jver-polish` - I'm not sure if that's intentional or not.

Yes, intentional. The "polish" PR is not approved yes.

File: src/NuGetGallery/Views/Pages/Contact.cshtml [hunk 1]
```diff
@@ -1,82 +1,101 @@
...
+            <p>
+                No need to contact support. Create a new user and transfer ownership of your packages to it.
+            </p>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-6">
+            <h3>Having issues working with NuGet.org?</h3>
+            <p>
+                Before reaching out to NuGet support, check out the <a href="https://docs.nuget.org/consume/nuget-faq#managing-packages-in-nuget.org">FAQ</a>.
```
Comment (microsoftsam) [inline src/NuGetGallery/Views/Pages/Contact.cshtml]: Mark the FAQ acronym with:
```<abbr title="Frequently Asked Questions">FAQ</abbr>```

File: src/Bootstrap/less/theme/base.less [hunk 2]
```diff
@@ -21,6 +21,17 @@ html {
...
+
+  .page-heading {
+    font-size: 2.5em;
+  }
+
+  .page-subheading {
+    font-weight: normal;
+    line-height: 0.5em;
+    color: #777;
+  }
```
Comment (microsoftsam) [inline src/Bootstrap/less/theme/base.less]: Good call.

Comment (microsoftsam) [inline src/Bootstrap/less/theme/base.less]: Alternatively, why not use the ms-font presets in fabric.css?

File: src/NuGetGallery/Views/Pages/Contact.cshtml (hunk 1 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Pages/Contact.cshtml]: Oh nice

File: src/Bootstrap/less/theme/base.less (hunk 2 above)
Comment (joelverhagen) [inline src/Bootstrap/less/theme/base.less]: I'll give this a try. 
http://dev.office.com/fabric#/styles/typography

Comment (joelverhagen) [inline src/Bootstrap/less/theme/base.less]: Yes, seems to work just fine. Nice level of control.

File: src/NuGetGallery/Views/Pages/Contact.cshtml (hunk 1 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Pages/Contact.cshtml]: Done.

File: src/NuGetGallery/Views/Pages/Contact.cshtml [hunk 3]
```diff
@@ -1,82 +1,101 @@
...
-    </p>
-
-    @if (User.Identity.IsAuthenticated)
-    {
-        <h3>The above doesn't help? Get in touch with us using the form below</h3>
-        <p>
-            A member of our Support team will be in touch with you soon.
-        </p>
-        using (Html.BeginForm("Contact", "Pages"))
+<article role="main" class="container main-container">
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Pages/Contact.cshtml]: Oops! nice catch

File: src/NuGetGallery/ExtensionMethods.cs [hunk 4]
```diff
@@ -310,17 +310,40 @@ public static MailAddress ToMailAddress(this User user)
             return html.TextBoxFor(expression, htmlAttributes);
         }
 
-        private static Dictionary<string, object> GetHtmlAttributes<TModel, TProperty>(HtmlHelper<TModel> html, Expression<Func<TModel, TProperty>> expression)
+        public static HtmlString ShowTextAreaFor<TModel, TProperty>(
```
Comment (joelverhagen) [inline src/NuGetGallery/ExtensionMethods.cs]: He will merge first. I will address merge conflicts.

Comment (loic-sharma) [inline src/NuGetGallery/ExtensionMethods.cs]: I've merged my changes, you can update this pull request with the latest version now.

**Pull request merged**